### PR TITLE
Fix type parameter in HistoryClient.show_history

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

File: bioblend/_tests/TestGalaxyHistories.py [hunk 1]
```diff
@@ -68,6 +68,22 @@ def test_show_history(self):
...
+        contents = self.gi.histories.show_history(self.history['id'], contents=True)
+        # Empty history has no datsets, content length should be 0
+        self.assertEqual(len(contents), 0)
+        self._test_dataset(history_id)
+        # history has 1 datset, content length should be 1
+        self.assertEqual(len(contents), 1)
+        contents = self.gi.histories.show_history(self.history['id'], contents=True, types=['dataset'])
+        # filtering for dataset, content length should still be 1
+        self.assertEqual(len(contents), 1)
+        contents = self.gi.histories.show_history(self.history['id'], contents=True, types=['dataset_collection'])
```
Comment (erasche) [inline bioblend/_tests/TestGalaxyHistories.py]: maybe add a test for `types=['dataset', 'dataset_collection']`?

**Comment by nsoranzo** [comment]
@mvdbeek I've added a commit updating some docs, I hope that's OK.

File: bioblend/galaxy/histories/__init__.py [hunk 2]
```diff
@@ -76,23 +76,28 @@ def show_history(self, history_id, contents=False, deleted=None, visible=None, d
         :param history_id: Encoded history ID to filter on
 
         :type contents: bool
-        :param contents: When ``True``, the complete list of datasets in the
-          given history.
+        :param contents: When ``True``, return also the complete list of
```
Comment (mvdbeek) [inline bioblend/galaxy/histories/__init__.py]: It's not `also`, it's a different output altogether.
```suggestion
        :param contents: When ``True``, return the complete list of
```

**Comment by mvdbeek** [comment]
Sure, no problem.

File: bioblend/galaxy/histories/__init__.py (hunk 2 above)
Comment (nsoranzo) [reply bioblend/galaxy/histories/__init__.py]: Uh, that's right! I've made it even more explicit in a rebase of the last commit.

**Pull request merged**